### PR TITLE
Added Artifact Registry

### PR DESCRIPTION
**Pull request by sergiogiuffrida**
Added GCP Artifact Registry support and customisable.

merge after https://github.com/jenkins-x/terraform-google-jx/pull/235

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1,20 +1,23 @@
 module "jx" {
-  source                          = "github.com/jenkins-x/terraform-google-jx?ref=v1.11.6"
+  source                          = "../terraform-google-jx"
```
Comment (msvticket) [inline main.tf]: Change to the released version

File: variables.tf [hunk 2]
```diff
@@ -173,3 +173,25 @@ variable "kuberhealthy" {
...
 }
 
+// GCP Artifact
+variable "artifact_location" {
+  description = "artifact registry repository Location"
+  type        = string
+  default     = "us-central1" 
+}
+variable "artifact_repository_id" {
+  description = "artifact registry repository Name, Defaul Cluster Name"
```
Comment (msvticket) [inline variables.tf]: Update description from terraform-google-jx. Don't forget to regenerate readme.

File: variables.tf [hunk 3]
```diff
@@ -173,3 +173,25 @@ variable "kuberhealthy" {
...
+  type        = string
+  default     = "oci"
+}
+variable "artifact_description" {
+  description = "artifact registry repository Description"
```
Comment (msvticket) [inline variables.tf:188]: Fix weird casing

**Review by msvticket — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 3 above)
Comment (sergiogiuffrida) [reply variables.tf:188]: Suggestion?

Comment (msvticket) [reply variables.tf:188]: Never mind. It should be fixed in the other descriptions and in terraform-google-jx as well. Some other time...

**Review by msvticket — APPROVED** [review]

**Pull request merged**